docs/dra/gpu-imex-pod.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2025 NVIDIA CORPORATION
+# SPDX-License-Identifier: Apache-2.0
+---
+apiVersion: resource.nvidia.com/v1beta1
+kind: ComputeDomain
+metadata:
+  name: imex-channel-injection
+spec:
+  numNodes: 1
+  channel:
+    resourceClaimTemplate:
+      name: imex-channel-0
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: gpu-imex-pod
+  labels:
+    runai/queue: test
+spec:
+  schedulerName: kai-scheduler
```

Comment:
@guptaNswati Do you think that we need to add node affinity here like in the example form  the gpu-operator that your referenced?

```
  affinity:
    nodeAffinity:
      requiredDuringSchedulingIgnoredDuringExecution:
        nodeSelectorTerms:
        - matchExpressions:
          - key: nvidia.com/gpu.clique
            operator: Exists

```

Reply:
mm i dont think its necessary for this simple example, especially if its tried on a GB200 node. This is more useful in a heterogeneous cluster